.github/workflows/ci.yml

```diff
@@ -385,15 +395,87 @@ jobs:
 
       - run: ${{ matrix.ls || 'ls -lh' }} dist/
 
-      - run: twine check --strict dist/*
```

Comment:
Why not do this check?

Reply:
twine wasn't installing on pypy 3.7, we also already do this check as part of the last release step, so technically redundant. I can probably restore after merging the build back into one step as per https://github.com/pydantic/pydantic-core/pull/678#discussion_r1238177524

Reply:
Restored this.